### PR TITLE
Implement /battle! command

### PR DESCRIPTION
**Pull request by panpawn**
#Zarel: we should have a /battle! command
#Zarel: complete with exclamation point
#Zarel: have it chastise you if you leave off the "!"

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -93,6 +93,12 @@ exports.commands = {
 		return target;
 	},
 
+	'battle!': 'battle',
+	battle: function (target, room, user, connection, cmd) {
+		if (cmd === 'battle') return this.sendReply("What?! How are you not more excited to battle?! Try /battle! to show me you're ready.");
+		return this.parse("/search randombattle");
```
Comment (Zarel) [inline commands.js]: tbh if arguments are actually passed, it should be identical to `/search`

**Comment by panpawn** [comment]
Is that the kind of thing you meant, @Zarel?

**Comment by Zarel** [comment]
I remember Cathy yelled at me when I had meme commands.

Now Cathy's not here to yell at me! :D

**Comment by ascriptmaster** [comment]
:+1:

**Pull request merged**